client/src/App.js

```diff
@@ -1,11 +1,96 @@
-import React, {useEffect, useState} from 'react';
-import logo from './logo.svg';
-import Ticket from './components/Ticket'
+import React, { useEffect, useState } from 'react';
+import axios from 'axios';
+import Ticket from './components/Ticket';
 import './App.css';
+import Search from './components/Search';
+import Header from './components/Header';
 
 function App() {
+  const [counter, setCounter] = useState(0); // counter of hidden tickets
+  const [tickets, setTickets] = useState([]); // list of tickets
+  const [call, setCall] = useState(0); // state that will be passed as a prop to restore tickets
+  const [scrollDisplay, setScrollDisplay] = useState('none'); // state that will be used to show scroll up button
+  const scrollFunction = () => { /* changing the display of the scroll up button if the
+    page scrolled down */
+    if (document.body.scrollTop > 20 || document.documentElement.scrollTop > 20) {
+      setScrollDisplay('block');
+    } else {
+      setScrollDisplay('none');
+    }
+  };
+  window.onscroll = scrollFunction;
+  const scrollUp = () => { // if the scroll up button pressed the page will scroll up
+    document.body.scrollTop = 0;
+    document.documentElement.scrollTop = 0;
+  };
+  useEffect(() => { // initial get request for all the tickets
```

Comment:
Why (re)define `fetch` if you only use it once and immediately?

Reply:
Because of console log error that react give if the function is not defined.